tasties_app/views.py

```diff
@@ -18,7 +18,7 @@ def recipes(request):
     recipes_with_ratings = {}
     for recipe in recipes_list:
         recipes_with_ratings[recipe] = Rating.objects.filter(recipe_id=recipe).aggregate(Avg('rating'))['rating__avg']
```

Comment:
But we found a much better method of querying the recipes, already sorted by rating, either ascending or descending. Please also notice that this current use of lambda functions DOES NOT ALLOW creating a new recipe without any ratings.